### PR TITLE
[neon] Just skip overly mocked tests

### PR DESCRIPTION
**Pull request by s0undt3ch**
### Previous Behavior
Remove this section if not relevant

### New Behavior
Remove this section if not relevant

### Tests written?
**[NOTICE] Bug fixes or features added to Salt require tests.**
Please review the [test documentation](https://docs.saltstack.com/en/latest/topics/tutorials/writing_tests.html) for details on how to implement tests into Salt's test suite.

Yes/No

### Commits signed with GPG?

Yes/No

Please review [Salt's Contributing Guide](https://docs.saltstack.com/en/latest/topics/development/contributing.html) for best practices.

See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information about signing commits with GPG.

### CODE REVIEW

File: tests/unit/utils/test_thin.py [hunk 1]
```diff
@@ -472,9 +468,9 @@ def test_gen_thin_python_exist_or_not(self):
                               'tops for Python 2',
                               handler.messages)
 
-    @skipIf(thin._six.PY2, reason='This test does too much patching on places it should not. Needs to be reworked')
+    @pytest.mark.skip('This test does too much patching on places it should not. Needs to be reworked')
```
Comment (Ch3LL) [inline tests/unit/utils/test_thin.py]: I really dont want this test to get lost. Can we add a similar message to the WARROOM and specify it needs to be re-worked? Or are we using this similar message for all tests that are over mocked?

**Review by Ch3LL — CHANGES_REQUESTED** [review]

**Review by dwoz — APPROVED** [review]

File: tests/unit/utils/test_thin.py (hunk 1 above)
Comment (s0undt3ch) [reply tests/unit/utils/test_thin.py]: We can append the WAR ROOM

Comment (s0undt3ch) [reply tests/unit/utils/test_thin.py]: Done

**Review by Ch3LL — APPROVED** [review]

**Review by twangboy — APPROVED** [review]

**Comment by dwoz** [comment]
Merging this without PR tests. It's just skipping tests.

**Pull request merged**